search/indexing.go

```diff
@@ -132,14 +133,30 @@ func (s *Server) indexProfile(ctx context.Context, ident *identity.Identity, rec
 	return nil
 }
 
-func (s *Server) updateUserHandle(ctx context.Context, did, handle string) error {
-	log := s.logger.With("repo", did, "op", "updateUserHandle", "handle", handle)
+func (s *Server) updateUserHandle(ctx context.Context, did syntax.DID, handle string) error {
+	log := s.logger.With("repo", did.String(), "op", "updateUserHandle", "handle_from_event", handle)
+
+	didID := syntax.AtIdentifier{Inner: did}
```

Comment:
the helper in https://github.com/bluesky-social/indigo/pull/385 could remove this line